.prettierignore

```diff
@@ -1 +1,2 @@
 js/flatbuffers.js
+js/errors.ts
```

Comment:
I think it's better to use `// prettier-ignore` in the js/errors.ts rather than ignoring the whole file. If it can't be turned on and off, then put the error codes at the bottom of the file

Reply:
@ry for some strange reasons `// prettier-ignore` does not have any effect.